style.css

```diff
@@ -77,11 +114,142 @@ a {
     height: 653.32px;
   }
 
-  .content {
+  .headline_content {
     margin-top: 17%;
   }
 
-  .action {
-    margin-top: 100px;
+  .works_content h1 {
+    padding: 0 30px;
+    margin-bottom: 25px;
+  }
+
+  .works_cards {
+    margin-top: 25px;
+    display: grid;
+    grid-template-columns: auto;
+    padding: 5%;
+    grid-template-rows: repeat(6, 475px);
+    gap: 20px;
+  }
+
+  .about_content {
+    background-image: url("Assets/about\ me-illustration-mobile.png");
+    background-repeat: no-repeat;
+    background-position: center;
+    height: 350.38px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    text-align: center;
   }
```

Comment:
- [ ] I appreciate😍😍 your effort in adding the `About` section to your project.
- [ ] However, it appears that the design of the section does not align with the suggested [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=0-1&t=xw0Xo7v32ZLKEBS9-0) design. 
- [ ] To ensure a professional look and feel for your project, I strongly recommend following the project requirements and adhering closely to the Figma design. This will help create a cohesive and polished project.🥱🥱🥱

Reply:
@AbdusSattar-70 Thankyou alot for the honest review, For the background of `about_content`, when I tried to export the image `illustration- about me- mobile` from Figma there seems to be an issue with this second frame of this illustration, I tried downloading it several times but was unable to download both the frames so this is the reason the background isn't matching a 100% with the template.